src/controllers/identity/policy/index.ts

```diff
@@ -30,9 +30,31 @@ const getPolicy = async (params) => {
 
 const listPolicies = async (params) => {
     const identityV1 = await grpcClient.get('identity', 'v1');
-    const response = await identityV1.Policy.list(params);
+    const identityResponse = await identityV1.Policy.list(params);
+    
+    /*1. Collect all repository_id
+    2. Collect repository.Policy with the set of repository_id
+    3. Merge repository.Policy into identity.Policy*/
+    const repositoryV1 = await grpcClient.get('repository', 'v1');
+    const repositories = await repositoryV1.Repository.list(params);
+    const repositoryIDList = repositories.results.map((d) => (d.repository_id));
+    const repositoryPromise = repositoryIDList.map((d) => { return repositoryV1.Policy.list({ ...params, repository_id: d }); });
+    const repositoryResponse = await Promise.allSettled(repositoryPromise);
 
-    return response;
+    const response: Array<any> = [];
+    identityResponse.results.map(d => response.push(d));
```

Comment:
```suggestion
    const responses: Array<any> = identityResponse.results.map(d => d)
```

array임을 암시하는 변수명이 좋을 것 같습니다.
map은 기본적으로 가공한 결과를 리턴하는 목적으로 사용하는 함수이므로, 위와 같이 사용하는 것이 가독성을 높이는 방법인 것 같아요.

Reply:
response 가 어떤 것인지 헷갈립니다.
policies 와 같이 이것이 담는 데이터가 무엇인지 명확하게 이름을 부여하면 어떨까요?